### ISSUE TITLE
Log Files not updating

### ISSUE THREAD

**Issue by ucfknight4life** [grokability/snipe-it#10277]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

My Storage\logs\larvel.log is not updating. It is just a blank file. I have enabled debug mode, however I believe the logs in debug are pulling from laravel.log so it is also blank. I am trying to troubleshoot an issue with the mail settings, but without the log files I am unsure where to start troubleshooting. Thank you,

### Reproduction steps

1. look at log files and they are empty
2. 
3.
...

### Expected behavior

log files should have information

### Screenshots

![image](https://user-images.githubusercontent.com/59296094/140509443-f14af57d-8472-42d6-a384-c85278a6679a.png)

### Snipe-IT Version

v5.3.0 build 6470 (gf87ffb84d)

### Operating System

Windows

### Web Server

IIS

### PHP Version

8.0

**Comment by snipe** [comment]
You'll want to make sure the permissions are set to be writable by the web server on your `storage` directory and all sub-directories and files within.

**Comment by ashhem** [comment]
Hi team,
I am facing a similar issue but with a slight deviation. My laravel.log file was created and was getting updated successfully. However, it reached a certain stage and has been stuck there without any updates for a while now. I am attaching the last 100 lines of the file here. I am actually trying to troubleshoot my mail server issue but because the file is stuck at this stage, I am not getting any new logs. I have already done the following things:

1. As per your suggestion here,
> You'll want to make sure the permissions are set to be writable by the web server on your `storage` directory and all sub-directories and files within.

Given the right permissions. In fact, since this not a rolled out production yet, I went all in and gave 777 permission to entire storage directory and it's contents (I agree this is not a good practice, just wanted to check if this is a permission issue). Although, since it was working well earlier, I do not think this may be a permission issue

[Snipe-IT Logs.txt](https://github.com/snipe/snipe-it/files/10682490/Snipe-IT.Logs.txt)

2.  I enabled the debugging mode and checked the logs on the toolbar too but the logs are stuck at the same stage and not printing anything new for the mail errors
3. Checked the https://snipe-it.readme.io/docs/common-issues page and cleared all possible cache(s) in order to start clean.

Other details:

1.  Hosted on a shared hosting running CentOS 7 Core via Apache (Document root controlled directly via CPanel)
2. Snipe-IT version is Version v6.0.14 - build 9038 (master)

Am I missing something here? Kindly let me know how to proceed in this case.
